src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
 export class Triangle {
+  shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: Figure['color'],
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side of triangle should by positive number!');
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error('You cannot build a triangle');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a)
+      * (s - this.b) * (s - this.c));
 
+    return Math.floor(area * 100) / 100;
```

Comment:
The getArea() methods should round the area down to hundredths, but currently it's rounding down to a whole number first and then dividing by 100. This can be fixed by rounding to hundredths first and then using floor.